### DISCUSSION TITLE
Mateksys r24 vario ELRS receiver channel 5 only 1 pin

### DISCUSSION THREAD

**Discussion by Capt-737NG** [ExpressLRS/ExpressLRS#2603] [Q&A]
Dear members, I want to use the channel 5 to control the landing gear (retract/extend) but there’s only one pin, while the LG servo has 3 pins (+ - S) which pin should I connect to the channel 5 on the receiver?

**Comment by mha1** [comment]
If you have a spare PWM connector use that an drive the landing gear with the respective mixer channel.

It now: your receiver should have a single pin marked 8. Use that to connect to the landing gear servo signal pin. Get + and - from one of the reveivers servo connector. The landing gear should now be controllable by EdgeTX mixer 8.

**Comment by Capt-737NG** [comment]
Thanks a lot Michael, I’ll do that! Cheers
Cristiano
